packages/menu/src/styles.ts

```diff
@@ -311,7 +311,7 @@ export const focusedMenuItemContainerStyle: Record<Theme, string> = {
     &:focus {
       text-decoration: none;
       background-color: ${palette.blue.light2};
-      color: ${palette.white};
```

Comment:
was darkmode broken previously?

Reply:
its hard to say, I worked on darkMode a few months ago and i remember going through all of this and had design QA everything so i'm not sure what happened

Reply:
kk